HW_Lesson_2/HW_Task_2.py

```diff
@@ -1 +1,10 @@
+eleks_employees = ["John", "Sara", "Jessica", "Andrew", "Mike"]
+toshiba_employees = ["Peter", "Jack", "Helen", "Mike"]
 
+# First option
+# toshiba_employees.extend(eleks_employees)
+# print(toshiba_employees)
```

Comment:
it is the correct option. append will add list as the last element.
Also after your manipulations, eleks should not have employees but it have. Please fix